keys.js

```diff
@@ -0,0 +1,13 @@
+var keys = {
+	twitterKeys : {
+	  consumer_key: '8lout56tMiVPh719vYkz3KfRU',
+	  consumer_secret: 'Cf4Lg8TwWHOfiPgmnRg5SYJLObLvFIF7b881jUMgPQb54MaCqZ',
+	  access_token_key: '41673945-pFtgLbRBmEAt2Xt7H4RhMNYuLJPVUHgGJnMY3LJXg',
+	  access_token_secret: 'bzTgNvR0KRHyOrXf40RKv4KDdKgwq4skORaIIPeL4L6nM',
+	},
+	spotifyKeys : {
+	  	client_id: '953cb5f705bb4e5084c876b0fab4d8a7',
+	  	client_secret: '2431345038374fb8ab1527bf1fd6eaca',
+	},
+}
+module.exports = keys;
```

Comment:
I would create the instances of your api's instead of making them in your liri file.